### PR TITLE
Support Renesas RA6 MCUs devices

### PR DESCRIPTION
**Pull request by duynguyenxa**
This PR target to support SoC and Board support as below:
EK-RA6M1
EK-RA6M2
EK-RA6M3
EK-RA6M4
EK-RA6M5
FPB-RA6E1
EK-RA6E2
FPB-RA6E2

Module:
SCI UART

### CODE REVIEW

**Review by ydamigos — CHANGES_REQUESTED** [review]
Please update maintainers yaml to add the board.

**Comment by quytranpzz** [comment]
> Please update maintainers yaml to add the board.

@ydamigos : I have updated the Renesas RA platform files to include the support of RA6 boards

File: soc/renesas/ra/ra6m5/Kconfig.soc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M5
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M5 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m5/Kconfig.soc:8]: help text indent is 1x tab followed by 2x spaces

File: soc/renesas/ra/ra6m5/Kconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6M5 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m5/Kconfig:17]:
```suggestion
```

File: boards/renesas/ek_ra6m5/doc/ek-ra6m5-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6m5/doc/ek-ra6m5-board.jpg:1]: Use webp and then put through https://tinypng.com/

File: soc/renesas/ra/ra6m3/Kconfig [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	select CPU_CORTEX_M4
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6M3 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m3/Kconfig:16]:
```suggestion
```

File: soc/renesas/ra/ra6m3/Kconfig.soc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M3
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M3 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m3/Kconfig.soc:8]: as before

File: boards/renesas/ek_ra6m3/doc/ek-ra6m3-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6m3/doc/ek-ra6m3-board.jpg:1]: as before

File: boards/renesas/ek_ra6m3/doc/index.rst [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6m3`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m3/doc/index.rst:105]: board target

File: boards/renesas/ek_ra6m5/doc/index.rst [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6m5`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m5/doc/index.rst:108]: board target

File: soc/renesas/ra/ra6e1/Kconfig [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6E1 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6e1/Kconfig:17]:
```suggestion
```

File: soc/renesas/ra/ra6e1/Kconfig.soc [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6E1
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6E1 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6e1/Kconfig.soc:8]: as before

File: boards/renesas/fpb_ra6e1/doc/fpb-ra6e1-board.jpg
Comment (nordicjm) [inline boards/renesas/fpb_ra6e1/doc/fpb-ra6e1-board.jpg:1]: as before

File: boards/renesas/fpb_ra6e1/doc/index.rst [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``fpb_ra6e1`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/fpb_ra6e1/doc/index.rst:92]: as before

File: soc/renesas/ra/ra6e2/Kconfig [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6E2 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6e2/Kconfig:17]:
```suggestion
```

File: soc/renesas/ra/ra6e2/Kconfig.soc [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6E2
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6E2 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6e2/Kconfig.soc:8]: as before

File: boards/renesas/ek_ra6e2/doc/ek-ra6e2-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6e2/doc/ek-ra6e2-board.jpg:1]: as before

File: boards/renesas/ek_ra6e2/doc/index.rst [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+
+.. figure:: ek-ra6e2-board.jpg
+	:align: center
+	:alt: RA6E2 Evaluation Kit
+
+	EK-RA6E2 Board Functional Area Definitions (Credit: Renesas Electronics Corporation)
+
+Hardware
+********
+Detail Hardware feature for the RA6E2 MCU group can be found at `RA6E2 Group User's Manual Hardware`_
```
Comment (nordicjm) [inline boards/renesas/ek_ra6e2/doc/index.rst:72]: detailed?

File: boards/renesas/ek_ra6e2/doc/index.rst [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+
+.. figure:: ra6e2-block-diagram.jpg
+	:width: 442px
+	:align: center
+	:alt: RA6E2 MCU group feature
+
+	RA6E2 Block diagram (Credit: Renesas Electronics Corporation)
+
+Detail Hardware feature for the EK-RA6E2 MCU can be found at `EK-RA6E2 - User's Manual`_
```
Comment (nordicjm) [inline boards/renesas/ek_ra6e2/doc/index.rst:81]: etc.

File: boards/renesas/ek_ra6e2/doc/index.rst [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6e2`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6e2/doc/index.rst:107]: board target

File: soc/renesas/ra/ra6m1/Kconfig [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+	select CPU_CORTEX_M4
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6M1 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m1/Kconfig:16]:
```suggestion
```

File: soc/renesas/ra/ra6m1/Kconfig.soc [hunk 16]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M1
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M1 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m1/Kconfig.soc:8]: as before

File: boards/renesas/ek_ra6m1/doc/ek-ra6m1-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6m1/doc/ek-ra6m1-board.jpg:1]: as before

File: boards/renesas/ek_ra6m1/doc/index.rst [hunk 17]
```diff
@@ -0,0 +1,159 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6m1`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m1/doc/index.rst:103]: as before

File: soc/renesas/ra/ra6m2/Kconfig [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+	select CPU_CORTEX_M4
+	select CPU_HAS_ARM_MPU
+	select HAS_RENESAS_RA_FSP
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6M2 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m2/Kconfig:16]:
```suggestion
```

File: soc/renesas/ra/ra6m2/Kconfig.soc [hunk 19]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M2
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M2 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m2/Kconfig.soc:8]: as before

File: boards/renesas/ek_ra6m2/doc/ek-ra6m2-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6m2/doc/ek-ra6m2-board.jpg:1]: as before

File: boards/renesas/ek_ra6m2/doc/index.rst [hunk 20]
```diff
@@ -0,0 +1,153 @@
...
+Detail Hardware feature for the RA6M2 MCU group can be found at `RA6M2 Group User's Manual Hardware`_
+
+.. figure:: ra6m2-block-diagram.jpg
+	:width: 871px
+	:align: center
+	:alt: RA6M2 MCU group feature
+
+	RA6M2 Block diagram (Credit: Renesas Electronics Corporation)
+
+Detail Hardware feature for the EK-RA6M2 MCU can be found at `EK-RA6M2 - User's Manual`_
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m2/doc/index.rst:71]: as before

File: boards/renesas/ek_ra6m2/doc/index.rst [hunk 21]
```diff
@@ -0,0 +1,153 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6m2`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m2/doc/index.rst:97]: as before

File: soc/renesas/ra/ra6m4/Kconfig [hunk 22]
```diff
@@ -0,0 +1,17 @@
...
+	select HAS_RENESAS_RA_FSP
+	select CLOCK_CONTROL_RENESAS_RA_CGC if CLOCK_CONTROL
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select FPU
+	select HAS_SWO
+	select XIP
+	help
+	  Enable support for Renesas RA6M4 MCU series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m4/Kconfig:17]:
```suggestion
```

File: soc/renesas/ra/ra6m4/Kconfig.soc [hunk 23]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA6M4
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA6M4 series
```
Comment (nordicjm) [inline soc/renesas/ra/ra6m4/Kconfig.soc:8]: as before

File: soc/renesas/ra/soc.yml [hunk 24]
```diff
@@ -1,7 +1,7 @@
 family:
   - name: renesas_ra
     series:
-      - name: ra4m1
+      - name: ra4m1_arduino
```
Comment (nordicjm) [inline soc/renesas/ra/soc.yml:4]: where is this changed in the Kconfig files?

File: boards/renesas/ek_ra6m4/doc/ek-ra6m4-board.jpg
Comment (nordicjm) [inline boards/renesas/ek_ra6m4/doc/ek-ra6m4-board.jpg:1]: as before

File: boards/renesas/ek_ra6m4/doc/index.rst [hunk 25]
```diff
@@ -0,0 +1,166 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra6m4`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/ek_ra6m4/doc/index.rst:110]: as before

File: boards/renesas/fpb_ra6e2/doc/fpb-ra6e2-board.jpg
Comment (nordicjm) [inline boards/renesas/fpb_ra6e2/doc/fpb-ra6e2-board.jpg:1]: as before

File: boards/renesas/fpb_ra6e2/doc/index.rst [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+
+.. figure:: fpb-ra6e2-board.jpg
+	:align: center
+	:alt: RA6E2 Fast Prototyping Board
+
+	FPB-RA6E2 Board Functional Area Definitions (Credit: Renesas Electronics Corporation)
+
+Hardware
+********
+Detail Hardware feature for the RA6E2 MCU group can be found at `RA6E2 Group User's Manual Hardware`_
```
Comment (nordicjm) [inline boards/renesas/fpb_ra6e2/doc/index.rst:61]: as before

File: boards/renesas/fpb_ra6e2/doc/index.rst [hunk 27]
```diff
@@ -0,0 +1,152 @@
...
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``fpb_ra6e2`` board configuration can be
```
Comment (nordicjm) [inline boards/renesas/fpb_ra6e2/doc/index.rst:94]: as before

**Review by nordicjm — DISMISSED** [review]

File: west.yml [hunk 28]
```diff
@@ -214,7 +214,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: af77d7cdfeeff290593e7e99f54f0c1e2a3f91e6
+      revision: pull/26/head
```
Comment (ydamigos) [inline west.yml:217]: PR was merged. Please use the latest hash.

**Comment by soburi** [comment]
Please update this PR title.

File: soc/renesas/ra/ra6m5/Kconfig.soc (hunk 1 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m5/Kconfig.soc]: I updated, and similar cases were also. Thanks

File: boards/renesas/ek_ra6m5/doc/ek-ra6m5-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6m5/doc/ek-ra6m5-board.jpg:1]: I changed to use .webp for the img file. Thanks

File: soc/renesas/ra/ra6m3/Kconfig (hunk 3 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m3/Kconfig:16]: Updated as above. Thanks

File: soc/renesas/ra/ra6m3/Kconfig.soc (hunk 4 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m3/Kconfig.soc]: Updated, thanks

File: boards/renesas/ek_ra6m3/doc/ek-ra6m3-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6m3/doc/ek-ra6m3-board.jpg:1]: Updated to use webp for img, thanks

File: boards/renesas/ek_ra6m3/doc/index.rst (hunk 5 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m3/doc/index.rst]: Updated, thanks.

File: boards/renesas/ek_ra6m5/doc/index.rst (hunk 6 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m5/doc/index.rst]: Same as above, I updated. thanks

File: soc/renesas/ra/ra6e1/Kconfig.soc (hunk 8 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6e1/Kconfig.soc]: Same as above, I updated. Thanks

File: boards/renesas/fpb_ra6e1/doc/fpb-ra6e1-board.jpg
Comment (quytranpzz) [reply boards/renesas/fpb_ra6e1/doc/fpb-ra6e1-board.jpg:1]: I updated to use the webp, thanks

File: boards/renesas/fpb_ra6e1/doc/index.rst (hunk 9 above)
Comment (quytranpzz) [reply boards/renesas/fpb_ra6e1/doc/index.rst]: I updated it, thanks.

File: soc/renesas/ra/ra6e2/Kconfig.soc (hunk 11 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6e2/Kconfig.soc]: I updated it, thanks.

File: boards/renesas/ek_ra6e2/doc/ek-ra6e2-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6e2/doc/ek-ra6e2-board.jpg:1]: I updated it, thanks.

File: boards/renesas/ek_ra6e2/doc/index.rst (hunk 12 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6e2/doc/index.rst]: Sorry for the typo. I updated it, thanks.

File: boards/renesas/ek_ra6e2/doc/index.rst (hunk 13 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6e2/doc/index.rst]: I updated it, thanks.

File: boards/renesas/ek_ra6e2/doc/index.rst (hunk 14 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6e2/doc/index.rst]: I updated it, thanks.

File: soc/renesas/ra/ra6m1/Kconfig.soc (hunk 16 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m1/Kconfig.soc]: I updated it, thanks.

File: boards/renesas/ek_ra6m1/doc/ek-ra6m1-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6m1/doc/ek-ra6m1-board.jpg:1]: I updated it, thanks.

File: boards/renesas/ek_ra6m1/doc/index.rst (hunk 17 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m1/doc/index.rst]: I updated it, thanks.

File: soc/renesas/ra/ra6m2/Kconfig.soc (hunk 19 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m2/Kconfig.soc]: I updated it, thanks.

File: boards/renesas/ek_ra6m2/doc/ek-ra6m2-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6m2/doc/ek-ra6m2-board.jpg:1]: I updated it, thanks.

File: boards/renesas/ek_ra6m2/doc/index.rst (hunk 20 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m2/doc/index.rst]: I updated it, thanks.

File: boards/renesas/ek_ra6m2/doc/index.rst (hunk 21 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m2/doc/index.rst]: I updated it, thanks.

File: soc/renesas/ra/ra6m4/Kconfig.soc (hunk 23 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m4/Kconfig.soc]: I updated it, thanks.

File: soc/renesas/ra/soc.yml (hunk 24 above)
Comment (quytranpzz) [reply soc/renesas/ra/soc.yml]: Sorry for the redundant change, this is a mistake. I removed this change. Thanks

File: boards/renesas/ek_ra6m4/doc/ek-ra6m4-board.jpg
Comment (quytranpzz) [reply boards/renesas/ek_ra6m4/doc/ek-ra6m4-board.jpg:1]: I updated it, thanks.

File: boards/renesas/ek_ra6m4/doc/index.rst (hunk 25 above)
Comment (quytranpzz) [reply boards/renesas/ek_ra6m4/doc/index.rst]: I updated it, thanks.

File: boards/renesas/fpb_ra6e2/doc/fpb-ra6e2-board.jpg
Comment (quytranpzz) [reply boards/renesas/fpb_ra6e2/doc/fpb-ra6e2-board.jpg:1]: I updated it, thanks.

File: west.yml (hunk 28 above)
Comment (quytranpzz) [reply west.yml]: I updated it, thanks

File: soc/renesas/ra/ra6m5/Kconfig (hunk 2 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m5/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6m3/Kconfig (hunk 3 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m3/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6e1/Kconfig (hunk 7 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6e1/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6e2/Kconfig (hunk 10 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6e2/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6m1/Kconfig (hunk 15 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m1/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6m2/Kconfig (hunk 18 above)
Comment (quytranpzz) [reply soc/renesas/ra/ra6m2/Kconfig]: I removed it. Thanks

File: soc/renesas/ra/ra6e1/soc.c [hunk 29]
```diff
@@ -0,0 +1,74 @@
...
+{
+	uint32_t key;
+
+	extern volatile uint16_t g_protect_counters[];
+
+	for (uint32_t i = 0; i < 4; i++) {
+		g_protect_counters[i] = 0;
+	}
+
+#if FSP_PRIV_TZ_USE_SECURE_REGS
```
Comment (nordicjm) [inline soc/renesas/ra/ra6e1/soc.c:46]: where is this defined?

Comment (quytranpzz) [reply soc/renesas/ra/ra6e1/soc.c:46]: Hi @nordicjm,
This macro was defined at https://github.com/zephyrproject-rtos/hal_renesas/blob/main/drivers/ra/fsp/src/bsp/mcu/all/bsp_common.h#L188-L192.

**Review by ydamigos — DISMISSED** [review]

**Review by thaoluonguw — DISMISSED** [review]

**Review by KhiemNguyenT — DISMISSED** [review]

**Review by ydamigos — DISMISSED** [review]

**Review by thaoluonguw — APPROVED** [review]

**Review by KhiemNguyenT — APPROVED** [review]

**Review by ydamigos — APPROVED** [review]

**Pull request merged**